wpeverest/everest-forms
# Dependency - EVF row conditional logic

### All Submissions:






### Changes proposed in this Pull Request:



This PR is dependent to [PR](https://github.com/wpeverest/everest-forms-pro/pull/566)

### How to test the changes in this Pull Request:

1. Create Form
2. Add Rows and Fields
3. Save Forms
4. Click on Setting Icon at top right of EVF row.
5. Add Conditional Logic
6. Preview Form and verify it works as expected.

### Types of changes:




### Other information:




### Changelog entry

> Dependency - EVF row conditional logic
